README.md

```diff
@@ -28,4 +28,6 @@ npm install -g truffle ganache-cli
 
 truffle compile
 truffle migrate
-```
\ No newline at end of file
+```
+
+MIT License
```

Comment:
Adding the MIT license is a good practice to clarify the terms of use for the project.